### DISCUSSION TITLE
How to use platform splash screen functionality on iOS without using MauiSplashScreen

### DISCUSSION THREAD

**Discussion by JoacimWall** [dotnet/maui#12967] [Q&A]
I'm not really clear on the status of this issue. Either way, this seems like more of a "how to" discussion than an actual issue in .NET MAUI. You definitely should be able to still use a custom splash screen.

Might be a nice candidate for an addition to the docs @davidbritch.

At this point I'm inclined to think that the functionality should work, but it's unclear how to do it. I think it might be better to open a discussion and/or maybe ask on a medium like Stack Overflow. 

If you do think something is actually not working and thus a bug with .NET MAUI, feel free to open a new issue.

Thanks!

_Originally posted by @jfversluis in https://github.com/dotnet/maui/issues/12917#issuecomment-1406700923_
 
Hi see my last comments. You say that it's just to do as we did before in Xamarin, that's not my experience, but it takes a little more to make it work according to my experience. It would be good if it was included in the documentation.

**Comment by davidortinau** [comment]
https://github.com/davidortinau/ControlGallery/blob/main/src/ControlGallery/Platforms/iOS/Info.plist#L14-L15

Add LaunchScreen.storyboard to your Platforms/iOS folder, and reference it in the info.plist. This overrides the MauiSplash storyboard that is added at build time. 

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/images/splashscreen?view=net-maui-7.0&tabs=ios

**Reply by hiten247** [reply]
This doesn't work. The custom LaunchScreen.storyboard should be added to the Maui Resources folder (at the root and not within Splash). 
 Next exclude the existing splash screen reference from the project file. Finally update the info.plist :
<key>UILaunchStoryboardName</key>
<string>LaunchScreen</string>

Running the app will now launch the storyboard launch screen.
